idaes/gas_distribution/properties/natural_gas.py

```diff
@@ -0,0 +1,397 @@
+#################################################################################
```

Comment:
To start a discussion, is this the correct place for these models? I.e. do we want these to be a new top-level folder in the directory structure, or should they be included under another folder?

One possibility would be to add `gas_distribution` as a sub folder under `generic_models/unit_models` for the unit operations, and to put the property packages in `generic_models/properties/examples`.